File: c#/program8.cs
using System;

class Pattern
{
    public int No;

    public Pattern(int no)
    {
        this.No = no;
    }

    public unsafe void DisplayPattern()
    {
        fixed(int *ptr = &No)

        for(int i = 1; i <= *(ptr); i++)
        {
            Console.Write("*\t");
        }
        Console.WriteLine();
    }
}

class program8
{
    public static void Main(string[] args)
    {
        int value = 0;

        Console.WriteLine("Enter frequency : ");
        value = int.Parse(Console.ReadLine());

        Pattern obj = new Pattern(value);
        obj.DisplayPattern();
    }
}

File: java/program8.java

import java.util.Scanner;

class Pattern
{
    public void Display(int iNo)
    {
        for(int iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("*\t");
        }
    }
}

class program8
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int iValue = sobj.nextInt();

        Pattern obj = new Pattern();
        obj.Display(iValue);
    }
}